.github/workflows/matrix.json

```
@@ -70,26 +89,26 @@
   },
   "full": {
     "static-checks": {
-      "php": [ "8.2" ],
-      "symfony": [ "^6.4" ],
+      "php": [ "8.2", "8.3" ],
+      "symfony": [ "^6.4", "^7.0" ],
       "api-platform": [ "^3.3" ]
     },
     "e2e-mariadb": {
-      "php": [ "8.2" ],
-      "symfony": [ "^6.4" ],
+      "php": [ "8.2", "8.3" ],
+      "symfony": [ "^6.4", "^7.0" ],
       "mariadb": [ "10.4.10" ],
       "state_machine_adapter": [ "winzou_state_machine", "symfony_workflow" ]
     },
     "e2e-mysql": {
-      "php": [ "8.2" ],
-      "symfony": [ "^6.4" ],
+      "php": [ "8.2", "8.3" ],
+      "symfony": [ "^6.4", "^7.0" ],
       "api-platform": [ "^3.3" ],
       "mysql": [ "5.7", "8.0" ],
       "twig": [ "^3.3" ]
     },
     "e2e-pgsql": {
-      "php": [ "8.2" ],
-      "symfony": [ "^6.4" ],
+      "php": [ "8.2", "8.3" ],
+      "symfony": [ "^6.4", "^7.0" ],
       "postgres": [ "13.9", "14.6" ]
```

```suggestion
      "postgres": [ "13.9", "14.13" ]
```
To be consistent, also we could add the latest 15 as well

---

We could bump 13 to the last minor and add also 16, but it could be done in a separate PR I guess, probably with some minor refactor of this matrix as there is probably no need for so many builds for all of these versions